guide/README.md

```diff
@@ -26,6 +26,18 @@ If you don't know JavaScript but would like to learn about it, here are a few li
 * [MDN's JavaScript guide and full documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
 * [Google, your best friend](https://google.com)
 
+### Useful tools
+Here are some helpful tools to know when getting started:
+
+* [Discord Developer Portal, create bots, manage tokens, calculate permissions](https://discord.com/developers/applications)
+* [Color picker, for the custom colors in embeds](https://www.color-hex.com/)
+* [Intents calculator](https://ziad87.net/intents/)
+* [Github Gists, to share code](https://gist.github.com/)
+* [Pastebin, to share code](https://pastebin.com/)
+* [Sourcebin, to share code](https://sourceb.in/)
+* [Starbin, to share code](https://starb.in/)
+* [StackOverflow, where developers learn](https://stackoverflow.com/)
```

Comment:
```suggestion
- [Discord Developer Portal](https://discord.com/developers/applications), create bots, manage tokens, calculate permissions
- [Color picker](https://www.color-hex.com/), for the custom colors in embeds
- [Intents calculator](https://ziad87.net/intents/), to calculate intent bitfields
- [Github Gists], [Pastebin](https://pastebin.com/), [Sourcebin](https://sourceb.in/), [Starbin](https://starb.in/), to share code
- [StackOverflow](https://stackoverflow.com/), where developers learn
```

Prettier usually turns `*` bullet points to `-`, so I thought I'll include that here. I've also made some other changes to clean it up a little bit, as I don't want it to take up too much space on the introduction page.